config.toml

```diff
@@ -96,11 +94,9 @@ weight = 1
 # section = ["HTML", "print"]
 
 [params]
-copyright_etcd = "The etcd Authors"
-copyright_linux = "Copyright © 2020 The Linux Foundation ®."
-# privacy_policy = "https://policies.google.com/privacy"
+copyright = "etcd Authors"
```

Comment:
Looking at the deploy preview and the footer changes, I think we still need the LF copyright line (as per https://github.com/cncf/foundation/blob/master/website-guidelines.md).

Reply:
Where does it say that we need to include an "LF copyright"?

Reply:
I reread the requirement more closely, and have withdrawn my comment.

Reply:
(The requirement has changed since last time I had to implement it, i should have read it more closely _before_ linking it. Sorry for the confusion)